sw/device/tests/sim_dv/pwrmgr_normal_sleep_all_wake_ups.c

```diff
@@ -0,0 +1,345 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/dif/dif_adc_ctrl.h"
+#include "sw/device/lib/dif/dif_pinmux.h"
+#include "sw/device/lib/dif/dif_pwrmgr.h"
+#include "sw/device/lib/dif/dif_rv_plic.h"
+#include "sw/device/lib/dif/dif_sensor_ctrl.h"
+#include "sw/device/lib/dif/dif_sysrst_ctrl.h"
+#include "sw/device/lib/irq.h"
+#include "sw/device/lib/runtime/ibex.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/testing/aon_timer_testutils.h"
+#include "sw/device/lib/testing/pwrmgr_testutils.h"
+#include "sw/device/lib/testing/rv_plic_testutils.h"
+#include "sw/device/lib/testing/test_framework/check.h"
+#include "sw/device/lib/testing/test_framework/ottf_main.h"
+#include "sw/device/lib/usbdev.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+#include "pwrmgr_regs.h"
+#include "sw/device/lib/testing/autogen/isr_testutils.h"
+
+/*
+  PWRMGR NORMAL SLEEP ALL WAKE UPS test
+
+  This test runs power manager wake up from deep sleep mode by
+  wake up inputs.
+
+  There are 6 wake up inputs.
+  0: sysrst_ctrl
+  1: adc_ctrl
+  2: pinmux
+  3: usb
+  4: aon_timer
+  5: sensor_ctrl
+
+ */
+
+#define kPinmuxWkupDetector5 5
```

Comment:
This is a C file, so declare this as a "static const int" or some such instead of a #define.